site/en/api_guides/python/index.md

```diff
@@ -1,52 +0,0 @@
-# Python API Guides
```

Comment:
This is an index file, not a list of ops... might want to keep, or add a redirect.

Reply:
I'll add a redirect to /guide
It doesn't look like you can access this page from: https://www.tensorflow.org/api_docs/python/tf ?

Reply:
cl/214344585